grow/testing/testdata/extract_localized/localized_pod/content/localized_collection/doc_in_parts.html

```diff
@@ -0,0 +1,22 @@
+---
+$localization:
+  locales:
+  - it
+  - ja
+  - ko
```

Comment:
Agree I’d missed a test case there — although I think your version removes a case we were previously testing. We could do:

```yaml
---
$localization:
  locales:
  - it
  - ja
  - sv
foo@: Localized base doc front matter in localized collection in localized pöd
bar@: Another localized base doc front matter in localized collection in localized pöd
baz@: Yet another localized base doc front matter in localized collection in localized pöd
---
{{_('Localized multipart doc base part body in localized collection in localized pöd')}}
---
$locale: ja
foo@: Localized doc part front matter in localized doc in localized collection in localized pöd
bar@: Another localized doc part front matter in localized doc in localized collection in localized pöd
---
{{_('Localized doc part body in localized doc in localized collection in localized pöd')}}
---
$locales:
- it
- sv
foo@: Multi-locale doc part front matter in localized doc in localized collection in localized pöd
---
{{_('Multi-locale doc part body in localized doc in localized collection in localized pöd')}}
```

- `Localized base doc front matter in localized collection in localized pöd` shouldn’t be extracted for _any_ locales (it’s overridden in all locales).
- `Another localized base doc front matter in localized collection in localized pöd` should only be extracted for `it` and `sv` (it’s overridden in `ja`).
- `Another localized doc part front matter in localized doc in localized collection in localized pöd` should only be extracted for `ja`.
- `Yet another localized base doc front matter in localized collection in localized pöd` should be extracted for `it`, `sv` and `ja` because it isn’t overridden at all.

— I think that covers the same case, without losing a case we were previously testing?